test_selectors.yml

```
@@ -4,6 +4,10 @@ selectors:
     selector: 11787463284727550157
   2337:
     selector: 12922642891491394802
+    name: geth-testnet-2
```

Should we call those instead?
```suggestion
    name: geth-devnet-2
```

---

@ajgrande924 thoughts?

---

that works lemme update